CODEOWNERS

```diff
@@ -55,14 +68,208 @@
 /cmake/sysbuild/suit_utilities.cmake      @nrfconnect/ncs-charon
 
 # All doc related files
-/doc/					  @nrfconnect/ncs-co-doc
+/doc/                                     @nrfconnect/ncs-co-doc
 /doc/_zoomin/                             @nrfconnect/ncs-co-doc @nrfconnect/ncs-doc-leads
-/doc/**/*.rst				  @nrfconnect/ncs-doc-owners
-/doc/**/*.svg				  @nrfconnect/ncs-doc-owners
-/doc/**/*.png				  @nrfconnect/ncs-doc-owners
-/doc/nrf/protocols/thread/*.rst           @wiba-nordic
-/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ @FrancescoSer
-/doc/nrf/app_dev/device_guides/nrf54l/    @annwoj
+/doc/kconfig/*.rst                        @nrfconnect/ncs-doc-leads
```

Comment:
Inside the `/doc/` folder you do not need the `*.rst` endings, just use:

`/doc/kconfig/`

and so on